changelog/fix_include_value_in.md

```diff
@@ -0,0 +1 @@
+* [#723](https://github.com/rubocop/rubocop-rails/pull/723): Fix include value in AttributeDefaultBlockValue. ([@kkitadate][])
```

Comment:
It seems better to use `Department/CopName` when describing the cop name.
Refs: https://github.com/rubocop/rubocop-rails/blob/master/CHANGELOG.md?plain=1#L9-L14

```markdown
* [#723](https://github.com/rubocop/rubocop-rails/pull/723): Fix include value in `Rails/AttributeDefaultBlockValue`. ([@kkitadate][])
```